doc/references.bib

```diff
@@ -823,6 +823,18 @@ @article{Jeurissen2011
   url       = {https://doi.org/10.1002/hbm.21032}
 }
 
+@article{Jones1999,
+  author    = {Jones, D K and Horsfield, M a and Simmons, a},
+  title     = {{Optimal strategies for measuring diffusion in anisotropic systems by magnetic resonance imaging.}},
```

Comment:
{{Optimal strategies for measuring diffusion in anisotropic systems by magnetic resonance imaging}},
```suggestion
  title     = {{Optimal strategies for measuring diffusion in anisotropic systems by magnetic resonance imaging}},
```